FinniversKit/Sources/Recycling/GridViews/AdRecommendations/Cell/StandardAdRecommendationCell.swift

```diff
@@ -165,7 +165,7 @@ public class StandardAdRecommendationCell: UICollectionViewCell, AdRecommendatio
 
     private func setup() {
         isAccessibilityElement = true
-        let accessibilityMultiplier: CGFloat = {
+        let accessibilityMultiplier: CGFloat = Config.isDynamicTypeEnabled ? {
```

Comment:
I think it might be a bit to complicated to read like this 😅 

How about simplifying with something like this
```
        var accessibilityMultiplier = 0.0
        if Config.isDynamicTypeEnabled {
            ...
        }
```